src/main/java/de/retest/recheck/review/ignore/matcher/ElementXPathMatcher.java

```diff
@@ -26,7 +29,7 @@ public boolean test( final Element element ) {
 
 	@Override
 	public String toString() {
-		return String.format( ElementXpathMatcherLoader.FORMAT, xpath );
+		return String.format( ElementXpathMatcherLoader.FORMAT, "/" + xpath );
```

Comment:
The leading slash is correct for the absolute XPath, but this doesn't correspond to our internal representation I guess? Is this good?

Reply:
I guess we should think about changing our internal representation. But this is definitely a separate PR...